modern/base/conan_bootstrap.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -ex
+
+version=`conan --version | awk '{print $3}'`
+
+if [[ "${version}" == 2.* ]]
+then
+    pip install conan==${CONAN_VERSION}
```

Comment:
The idea was "eating our own dog food", by installing some tools from Conan Center, instead of from APT repository. Here we have binutils installed in /opt. 

So, you can use this bootstrap, but it needs to install for both versions.

Reply:
I would suggest reverting it and using only Conan 1.x to install binutils package with lockfile for now. We can update it in the future, when dropping Conan 1.x support.

Reply:
I tested again after reverting these changes and it worked.  I don't know what happened with this step the first time ...

Reply:
By default, it will have Conan 1.x installed, then it installs binutils. As last step, we install CONAN_VERSION. To make it compatible with Conan 2.x would require a new profile file, a new lockfile, a new command line. Too much and not needed for now.